packages/@misk/core/tests/features/Navbar/__snapshots__/Container.test.tsx.snap

```diff
@@ -154,31 +128,36 @@ exports[`Misk Navbar Container Container can render 1`] = `
 }
 
 <div
-    class="bp3-navbar emotion-10"
+    class="bp3-navbar emotion-9"
   >
     <div
-      class="emotion-2"
+      class="emotion-1"
     >
       <div
-        class="bp3-navbar-group bp3-align-left bp3-dark emotion-1"
+        class="bp3-navbar-group bp3-align-left bp3-dark"
+        css="You have tried to stringify object returned from \`css\` function. It isn't supposed to be used directly (e.g. as value of the \`className\` prop), but rather handed to emotion so it can handle it (e.g. as value of \`css\` prop)."
       >
         <span
           class="emotion-0"
         >
           Misk
         </span>
       </div>
+      <div
+        class="bp3-navbar-group bp3-align-left bp3-dark"
+        css="You have tried to stringify object returned from \`css\` function. It isn't supposed to be used directly (e.g. as value of the \`className\` prop), but rather handed to emotion so it can handle it (e.g. as value of \`css\` prop)."
```

Comment:
This is interesting and unexpected.

Reply:
Fixed! Forgot to do [this](https://emotion.sh/docs/css-prop#import-the-jsx-function-from-emotioncore).